### DISCUSSION TITLE
Where are the definitions for $300-$344?

### DISCUSSION THREAD

**Discussion by jrdarrah** [terjeio/grblHAL#282] [Q&A]
I found this -- https://github.com/terjeio/grblHAL/blob/master/doc/markdown/settings.md
but it doesn't describe the fields starting with $300.  I'm in the process of getting a CNC mill conversion done and I'm at the point where I want to go line by line through all of the  $xxx commands. Is there a comprehensive list of all of the grblHAL commands?

**Comment by phil-barrett** [comment]
In the [Wiki](https://github.com/terjeio/grblHAL/wiki/Additional-or-extended-settings---Copy).

**Comment by terjeio** [comment]
From a terminal window you may also use:

`$help settings` for a list of all settings with a short description
`$help` for a list of help arguments
`$$=<n>` where `<n>` is setting number for a short description of it (same info as for `$help settings`)
`$<n>` for the value of setting `<n>`
